### PR TITLE
Re-balances quirk values to be less impossible to nuance

### PR DESCRIPTION
**Pull request by ghost**
:cl: Qustinnus
tweak: All quirk point values have been re-done
/:cl:

negative quirks were impossible to create because there was zero nuance between how bad traits were, for example if you got light drinker and family heirloom, two relatively easy quirks to manage, you can get freerunner. This meant a ton of neutral quirks had to be neutral quirks because giving them even -1 points would make it too easy to get good traits. I've bumped up most quirks by x4 to allow for more nuance between what quirks are actually worth. With some of them being lowered (Like family heirloom and light drinker) as theyre not really supposed to be worth so much for so little.

quick log of changes for CB:
- Every quirk not mentioned had it's point value increased x4 (e.g. 1 -> 4, 4 -> 16)
- mime / clown fan is 2 points (change of -50% with this pr)
- musician is 2 points (change of -50% with this pr)
- depression goes from -1 to -3 points (change of -25% with this pr)
- nyctophobiagoes from -1 to -3 points (change of -25% with this pr)
- junkie from -2 to -6 points (change of 25% with this pr)
- frail from -2 to -6 points (change of 25% with this pr)
- smoker from -2 to -6 points (change of 25% with this pr)
- unstable -2 to -10 points (change of 25% with this pr)

### CODE REVIEW

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -630,7 +630,7 @@
 /datum/quirk/junkie/smoker
 	name = "Smoker"
 	desc = "Sometimes you just really want a smoke. Probably not great for your lungs."
-	value = -1
+	value = -2
```
Comment (Iamgoofball) [inline code/datums/traits/negative.dm:633]: :-1: make this a 4

File: code/datums/traits/negative.dm [hunk 2]
```diff
@@ -43,7 +43,7 @@
 /datum/quirk/blindness
 	name = "Blind"
 	desc = "You are completely blind, nothing can counteract this."
-	value = -4
+	value = -15
```
Comment (ATH1909) [inline code/datums/traits/negative.dm:46]:
```suggestion
	value = -16
```

File: code/datums/traits/good.dm [hunk 3]
```diff
@@ -62,7 +62,7 @@
 /datum/quirk/fan_clown
 	name = "Clown Fan"
 	desc = "You enjoy clown antics and get a mood boost from wearing your clown pin."
-	value = 1
+	value = 4
```
Comment (RaveRadbury) [inline code/datums/traits/good.dm:65]: You've set clown fan to 4 and mime fan to 2. These need to be equal.

**Comment by optimumtact** [comment]
what hellish bikeshed have you unleashed now

**Comment by Qustinnus** [comment]
psy-ops to distract them from our actual relevant PRs

File: code/datums/traits/good.dm (hunk 3 above)
Comment (RaveRadbury) [inline code/datums/traits/good.dm:65]:
```suggestion
	value = 2
```

Comment (RaveRadbury) [reply code/datums/traits/good.dm:65]: This would bring it in line with the values you set for Mime Fan (its counterpart) and Musician.

**Comment by Timberpoes** [comment]
Let's push some buttons and see what happens.

**Review by Timberpoes — DISMISSED** [review]

**Comment by Timberpoes** [comment]
Actually double-checking the to-do list from the PR body before I merge this. Pinged you on Discord.

File: code/datums/traits/good.dm [hunk 4]
```diff
@@ -182,21 +182,21 @@
...
-	value = 2
+	value = 8
 	mob_trait = TRAIT_SKITTISH
 	medical_record_text = "Patient demonstrates a high aversion to danger and has described hiding in containers out of fear."
 
 /datum/quirk/spiritual
 	name = "Spiritual"
 	desc = "You hold a spiritual belief, whether in God, nature or the arcane rules of the universe. You gain comfort from the presence of holy people, and believe that your prayers are more special than others."
-	value = 1
+	value = 4
```
Comment (Jared-Fogle) [inline code/datums/traits/good.dm:199]: this straight up isn't worth 4 points, i don't know any admin that treats these differently honestly

**Comment by optimumtact** [comment]
dear god the bikeshed has already begun and it's not even merged

File: code/datums/traits/good.dm (hunk 4 above)
Comment (Qustinnus) [reply code/datums/traits/good.dm:199]: this pr does not change the status quo of this quirk, so it doesn't matter.

Comment (Jared-Fogle) [reply code/datums/traits/good.dm:199]: fair enough

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Do me a favor and just make a .md chart of the old/new values in the desc. Reviewing this hurty.

File: code/modules/client/preferences_savefile.dm [hunk 5]
```diff
@@ -488,6 +488,7 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 			job_preferences -= j
 
 	all_quirks = SANITIZE_LIST(all_quirks)
+	validate_quirks()
```
Comment (Cyberboss) [inline code/modules/client/preferences_savefile.dm:491]: Since when is this necessary?

Comment (Cyberboss) [reply code/modules/client/preferences_savefile.dm:491]: Nevermind, I get it, it's for savefile migration

**Review by Cyberboss — APPROVED** [review]
Seems reasonable

**Pull request merged**